src/features/Applet/Schedule/Calendar/YearView/MonthCalendar/CalendarDate/CalendarDate.tsx

```diff
@@ -110,7 +115,9 @@ export const CalendarDate = ({ dateToRender, dateOfMonth, onClick, events }: Cal
           </StyledTooltipDate>
           {getEventsRows()}
           {showMoreText && (
-            <StyledMore>{`${events.length - 9} ${t('more').toLowerCase()}...`}</StyledMore>
+            <StyledMore>{`${events.length - MAX_EVENTS_IN_TOOLTIP} ${t(
+              'more',
+            ).toLowerCase()}...`}</StyledMore>
```

Comment:
Maybe it's better to create a constant for this?

Reply:
done